support/cas-server-support-redis-core/src/test/java/org/apereo/cas/redis/core/RedisObjectFactoryTests.java

```diff
@@ -68,6 +68,18 @@ void verifyConnectionWithUsernamePassword() throws Throwable {
         assertNotNull(connection);
     }
 
+    @Test
+    void verifyConnectionWithUsernamePasswordOverTls() throws Throwable {
+        val props = new BaseRedisProperties();
+        props.setHost("localhost");
+        props.setPort(16669);
+        props.setUsername("default");
+        props.setPassword("pAssw0rd123");
+        props.setUseSsl(true);
+        val connection = RedisObjectFactory.newRedisModulesCommands(props);
+        assertNotNull(connection);
```

Comment:
Please update this line to make sure a connection can in fact be opened:

```
        assertNotNull(connection);
        try (val con = connection.getConnection()) {
            con.select(0);
        }
```

Your test needs to look more like this:

https://github.com/apereo/cas/blob/master/support/cas-server-support-redis-core/src/test/java/org/apereo/cas/redis/core/RedisObjectFactoryTests.java#L69-L86